EIPS/eip-4844.md

```diff
@@ -307,14 +340,10 @@ def get_intrinsic_gas(tx: SignedBlobTransaction, pre_state: ExecState) -> int:
     intrinsic_gas += len(tx.message.blob_versioned_hashes) * get_blob_gas(pre_state)
     return intrinsic_gas
 
-def get_blob_gas(pre_state: ExecState) -> int:
+def get_blob_gas(header: Header) -> int:
     blocks_since_start = get_current_block(pre_state) - FORK_BLKNUM
     targeted_total = blocks_since_start * TARGET_BLOB_TXS_PER_BLOCK
-    actual_total = read_storage(
-        pre_state,
-        SYSTEM_STATE_ADDRESS,
-        TOTAL_BLOB_TXS_STORAGE_SLOT
-    )
+    actual_total = targeted_total + header.blob_txs
```

Comment:
Please note that I've updated the gas calculation to sum the running excess of blob txs and the targeted total value.